.teamcity/_self/projects/WPComTests.kt

```diff
@@ -220,13 +225,14 @@ fun gutenbergBuildType(screenSize: String, buildUuid: String): BuildType {
 	}
 }
 
-fun gutenbergPlaywrightBuildType( targetDevice: String, buildUuid: String, atomic: Boolean = false ): E2EBuildType {
+fun gutenbergPlaywrightBuildType( targetDevice: String, buildUuid: String, atomic: Boolean = false, edge: Boolean = false ): E2EBuildType {
```

Comment:
I think at some point we'll probably want to create an `options` object for all these optional parameters, so we can avoid a bunch of label-less booleans when calling. But we don't have to do that in this PR! 👍

Reply:
I also like the idea of using some sort of options object to make it more clear when you call `gutenbergPlaywrightBuildType` what exact build we're trying to run. Otherwise, you have to track down the true/false statements and find what they mean :)

Reply:
Sure, I can implement the changes for this PR. I initially avoided doing so since we're only 2 parameters in, but I do agree it is unsustainable once additional params are added.

With that said though, at 3+ params we might be better off splitting the function into smaller pieces.

Reply:
I tried for a bit to get something akin to a TypeScript interface going, but I ended up going back to a named parameter approach instead as it was simple and required no changes to the current implementation.